conda/libmesh-vtk/build.sh

```diff
@@ -18,11 +18,13 @@ cmake .. -G "Ninja" \
     -DVTK_BUILD_TESTING:BOOL=OFF \
     -DVTK_BUILD_EXAMPLES:BOOL=OFF \
     -DBUILD_SHARED_LIBS:BOOL=ON \
-    -DVTK_USE_MPI:BOOL=ON \
     -DVTK_GROUP_ENABLE_Rendering:STRING=DONT_WANT \
     -DVTK_GROUP_ENABLE_Qt::STRING=NO \
     -DVTK_GROUP_ENABLE_Views:STRING=NO \
-    -DVTK_GROUP_ENABLE_Web:STRING=NO 
+    -DVTK_GROUP_ENABLE_Web:STRING=NO \
+    -DVTK_GROUP_ENABLE_MPI=NO \
+    -DVTK_USE_MPI:BOOL=OFF \
```

Comment:
why this change?

Reply:
Latest MPICH mucked with this somehow. And its how these errors were cropping up all over the place:

```pre
kernels/hfem.dirichlet: 2024-01-22 06:51:22.496 (   0.059s) [         26216F0] vtkMPICommunicator.cxx:67    WARN| MPI had an error
kernels/hfem.dirichlet: ------------------------------------------------
kernels/hfem.dirichlet: Invalid datatype, error stack:
kernels/hfem.dirichlet: internal_Bcast(107): MPI_Bcast(buffer=0x16b46115c, count=1, MPI_DATATYPE_NULL, 0, comm=0x84000004) failed
kernels/hfem.dirichlet: internal_Bcast(67).: Datatype for argument datatype is a null datatype
kernels/hfem.dirichlet: ------------------------------------------------
kernels/hfem.dirichlet: Abort(526462467) on node 0 (rank 0 in comm 432): application called MPI_Abort(comm=0x84000004, 526462467) - process 0
```

Doing some research on this error turned up rather dull one-offs. But it Kitware blames this error on users not using the same version of MPI as it was built with. Of course, that was not our case. But it was the only reference I could find on the internets.

Reply:
The line right below your comment:

`-DVTK_MODULE_ENABLE_VTK_IOMPIParallel=WANT`

We're still using MPI. But only for the necessary FileWriter we are using in libMesh (or so John Peterson required back in the day).

Reply:
sounds good